WooCommerce/Classes/POS/Presentation/POSFloatingControlView.swift

```diff
@@ -93,14 +96,14 @@ private extension POSFloatingControlView {
 @available(iOS 17.0, *)
 extension POSFloatingControlView {
     static var secondaryFontColor: Color {
-        return .posDarkGray.opacity(0.6)
+        .posOnDisabledContainer
```

Comment:
🗒️ Note for reviewer: there is no design for the disabled state (I tried asking about it in 1qcjzXitBHU7xPnpCOWnNM-fi-18_5636#1121088204 but maybe there was some misunderstanding and I didn't get a final answer), so I picked the disabled colors in the design system (there are only two).